### ISSUE TITLE
Jave heap space on invite

### ISSUE THREAD

**Issue by paulodamaso** [zerocracy/farm#1206]
Tried to make an invitation, zerocrat had an unrecoverable failure:

```
PID: 4@ec9929b8-b78d-47e8-b6c3-2bf0ccff1137, thread: zerocrat_bot Telegram Executor
java.util.Arrays[3664] java.lang.OutOfMemoryError: Java heap space
```

0.22.28 (https://github.com/zerocracy/farm): ID: 344195911, From: pauloeduardolobo, ChatId: 227326096

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@paulodamaso thanks, it was fixed in #1208

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#1206` is not assigned, can't get performer
